src/main/java/de/picturesafe/search/elasticsearch/connect/mapping/MappingBuilder.java

```diff
@@ -96,6 +97,10 @@ private void addFields(XContentBuilder mapping, List<? extends FieldConfiguratio
                 } else {
                     mapping.field("type", fieldConfig.getElasticsearchType());
                 }
+                if (fieldConfig.getElasticsearchType().equalsIgnoreCase(ElasticsearchType.OBJECT.toString())
```

Comment:
Why only type object?

Reply:
See: https://www.elastic.co/guide/en/elasticsearch/reference/current/enabled.html
"The enabled setting, which can be applied only to the top-level mapping definition and to object fields"

Reply:
And: "Note that because Elasticsearch completely skips parsing the field contents, it is possible to add non-object data to a disabled field"